lib/Song.rb

```diff
@@ -0,0 +1,74 @@
+require 'pry'
+
+class Song
+
+  attr_accessor :name, :artist, :genre
+
+  @@all = []
+
+  def initialize(name, artist=nil, genre=nil)
```

Comment:
Nit: spaces between ` = `. It is a rubocop offense to have these right up next to each other.